src/AppBundle/Service/Packagist/PackagistServiceProvider.php

```diff
@@ -81,14 +85,18 @@ private function callApi($packageName)
         $externalData = $this->packagistClient->get($packageName);
         $packageDetails['maintainers'] = $this->excludeMaintainers($externalData->getMaintainers());
         $packageDetails['authorAvatarUrl'] = $this->getAuthorAvatarUrl($externalData->getRepository());
+        $packageDetails['description'] = $externalData->getDescription();
+        $packageDetails['downloads'] = $externalData->getDownloads()->getTotal();
+        $packageDetails['forks'] = $externalData->getGithubForks();
+        $packageDetails['stars'] = $externalData->getGithubStars();
         $versions = $externalData->getVersions();
         if (is_array($versions) && !empty($versions)) {
             if (isset($versions['dev-master'])) {
                 $version = 'dev-master';
             } else {
                 $version = key($versions);
             }
-            $packageDetails['updated'] = $versions[$version]->getTime();
+            $packageDetails['updated'] = \DateTime::createFromFormat(\DateTime::ISO8601, $versions[$version]->getTime());
```

Comment:
Yes, we need timestamp but Packagist provides date and time in format: [ISO8601](http://php.net/manual/pl/class.datetime.php#datetime.constants.iso8601). I think the safest way is to create DateTime object from given date and time and then use it as timestamp using `format('U')`. See: [https://github.com/ezsystems/ezplatform-com/pull/101/files#diff-c9656b10c2a76d5eb42d06387e7ba085R87](https://github.com/ezsystems/ezplatform-com/pull/101/files#diff-c9656b10c2a76d5eb42d06387e7ba085R87)